.storybook/stories/label/label.stories.ts

```diff
@@ -78,10 +102,76 @@ export const ColorLabelClosable: StoryObj = {
 export const StatusLabelClickableWithClose: StoryObj = {
   argTypes: {
     labelType: { control: { disable: true }, table: { disable: true } },
+    showProjectedContent: { control: { disable: true }, table: { disable: true } },
+    cssLabel: { control: { disable: true }, table: { disable: true } },
   },
   args: {
     labelTypes: LABEL_STATUS_TYPES,
     clickable: true,
     closeIcon: true,
   },
 };
+
+export const AngularComponent: StoryObj = {
```

Comment:
I think LabelComponent will fit better on the story instead of seeing Angular ..., Angular..., Angular.. as story items.
```suggestion
export const LabelComponent: StoryObj = {
```

Reply:
It's good to have a story with a single label component and then have stories with all the types available.